test/expenses.routes.test.js

```diff
@@ -829,8 +831,7 @@ describe('expenses.routes.test.js', () => {
     let group, user;
     return (g ? Promise.resolve(g) : models.Group.create(utils.data('group2')))
       .tap(g => group = g)
-      .then(() => u ? u : models.User.create(utils.data('user4'))
-        .tap(user => group.addUserWithRole(user, roles.HOST)))
+      .then(() => u ? u : expenseFiler)
```

Comment:
Not a fan of this function. `createExpense` should not be responsible to create a group. Also, this particular line is very hard to follow. Why is a user turning into `expenseFiler`?